src/shared/api/dataProvider.tsx

```diff
@@ -35,37 +41,37 @@ const dataProvider: DataProvider = {
                     if (status < 200 || status >= 300) {
                         return reject(
                             new HttpError(
-                                (json && json.message) || statusText,
+                                (json && json.message) || body,
                                 status,
-                                json
-                            )
+                                json,
+                            ),
                         );
                     }
                     return resolve({
                         data: json.items,
                         total: json.meta.total,
                         pageInfo: {
                             hasNextPage: json.meta.has_next,
-                            hasPreviousPage: json.meta.has_previous
-                        }
+                            hasPreviousPage: json.meta.has_previous,
+                        },
                     });
                 });
         });
     },
     getOne: (resource, params) => {
         const connectionId = params.id;
         return new Promise((resolve, reject) => {
-            return fetch(
-                `${apiUrl}/v1/${resource}/${connectionId}`,
-                getHeader()
-            )
+            return fetch(`${apiUrl}/v1/${resource}/${connectionId}`, {
```

Comment:
```suggestion
            return fetch(`${apiUrl}/v1/${resource}/${id}`, {
```
Этот метод ведь общий на все типы сущностей, не только connection